arcade/sprite/sprite.py

```diff
@@ -313,7 +313,13 @@ def stop(self) -> None:
 
     # ---- Draw Methods ----
 
-    def draw(self, *, filter=None, pixelated=None, blend_function=None) -> None:
+    def draw(
+            self,
+            *,
+            filter: Optional[OpenGlFilters] | None = None,
```

Comment:
```suggestion
            filter: Optional[OpenGlFilters] = None,
```
Although the `|` syntax is shorter, it's not appropriate here:
1. Arcade has to support 3.8+, but `|` is a 3.10+ feature.
2. You've already used `Optional` to mark the possible `None` value.

Reply:
yeah sorry, I forgot to remove the `| None` part